src/target/source/codegen_cuda.cc

```diff
@@ -591,13 +591,17 @@ void CodeGenCUDA::VisitExpr_(const RampNode* op, std::ostream& os) {
 }
 
 void CodeGenCUDA::VisitExpr_(const BroadcastNode* op, std::ostream& os) {   // NOLINT(*)
-  if (op->dtype.is_int() && op->dtype.bits() == 8 && op->lanes == 4) {
+  if ((op->dtype.is_int() || op->dtype.is_uint()) && op->dtype.bits() == 8 && op->lanes == 4) {
     // make_int8x4
     const int64_t *p = as_const_int(op->value);
     CHECK(p);
     int64_t v = *p & 0xFF;
     v = (v << 24) | (v << 16) | (v << 8) | v;
-    os << "(int)" << v;
+    if (op->dtype.is_uint()) {
```

Comment:
why do we care the signedness? this just downcasts to 32 bits,.

Reply:
TVM uses uint to store unit8x4 (in function PrintType). The care will generate code like `unit x = (unit)y`, instead of `unit x = (int)y`. And what is your further opinion?

Reply:
Can we keep it as is? I do not see benefits from this change. Otherwise the entire PR LGTM. Thanks!

Reply:
I think it's not necessary to revert this change, if it's harmless. Consider that  `CodeGenCUDA::PrintType` for uint8x4 generates "uint", this change somehow makes sense.